### PR TITLE
consent form, login, and summary updated for displacement study

### PR DESCRIPTION
**Pull request by jenny3248**
Sorry, I sent the last pull request to master.

### CODE REVIEW

File: www/templates/intro/login.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
 <center><h3>Login</h3></center></div>
 
 <div class="intro-text">
-Please log-in with the code provided to you by a UC Berkeley Researcher.
+Please log-in with the email provided to you by a UC Berkeley Researcher.
```
Comment (shankari) [inline www/templates/intro/login.html]: Shouldn't this still be `code`? At least looking at the consent document, you are not going to use emails.

Comment (jenny3248) [reply www/templates/intro/login.html]: Emails is correct thank you

File: www/templates/intro/consent.html [hunk 2]
```diff
@@ -1,35 +1,50 @@
...
+
+In the app, you will then be asked to log-in using a unique ID number that will be associated with the data in place of your name. Once you have joined the User Interface for this study, the locational data from your smartphone device will be automatically transferred to the research team’s private, protected server when you are connected to Wifi. You will have E-Mission downloaded on your phone for at least the four weeks in between the first and second visit.
```
Comment (shankari) [inline www/templates/intro/consent.html]: But that is consistent with this statement from the consent document, no?

**Comment by shankari** [comment]
@jenny3248 deployed and live.

**Pull request merged**